### DISCUSSION TITLE
CipherOS uploading issue not fixed

### DISCUSSION THREAD

**Discussion by ItsAdityan** [ROM-builders/temporary#8650] [Q&A] [answered]
https://cirrus-ci.com/task/5790479718547456

**Comment by Apon77** [comment, marked as answer]
> https://cirrus-ci.com/task/5790479718547456

It's because of your this build, https://cirrus-ci.com/build/5685338986250240
you started this 1 hour before I fixed the uploading things. That's why it didn't get fixed. 
Actually, the problem is fixed. 
Anyway, files are uploaded. Check the download website.
Thank you.

**Reply by ItsAdityan** [reply]
Thanks
